endpoints/openrtb2/amp_auction_test.go

```diff
@@ -476,3 +718,75 @@ func (m *mockAmpExchange) HoldAuction(ctx context.Context, bidRequest *openrtb.B
 
 	return response, nil
 }
+
+func getTestBidRequest(nil_user bool, nil_ext bool, consent_string string, digitrust_ID string) ([]byte, error) {
```

Comment:
Please change the parameter names to be in camel case per Go's naming conventions

Reply:
done